### PR TITLE
[zh-cn] sync coarse-parallel-processing-work-queue.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 1]
```diff
@@ -129,114 +134,133 @@ Waiting for pod default/temp-loe07 to be running, status is Pending, pod ready:
...
+```
+foo
+```
 
-root@temp-loe07:/# /usr/bin/amqp-publish --url=$BROKER_URL -r foo -p -b Hello
+<!--
+Publish one message to the queue:
+# And get it back.
+-->
+向它推送一条消息：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md:238]:
```suggestion
向队列推送一条消息：

```

File: content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 2]
```diff
@@ -249,7 +273,7 @@ In a practice, the content of the messages might be:
 -->
 ## 为队列增加任务  {#filling-the-queue-with-tasks}
 
-现在让我们给队列增加一些任务。在我们的示例中，任务是多个待打印的字符串。
+现在用一些模拟任务填充队列。在我们的示例中，任务是多个待打印的字符串。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md:276]:
```suggestion
现在用一些模拟任务填充队列。在此示例中，任务是多个待打印的字符串。
```

File: content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 3]
```diff
@@ -260,47 +284,60 @@ In a practice, the content of the messages might be:
...
+# 在你的计算机上运行此命令，而不是在 Pod 中
+/usr/bin/amqp-declare-queue --url=$BROKER_URL -q job1  -d
+```
+```
+job1
+```
+<!--
+Add items to the queue:
+-->
+将项目添加到队列中：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md:315]:
```suggestion
将这几项添加到队列中：
```

**Review by Princesso — COMMENTED** [review]
@sftim could you please have a look at this PR? There seems to be an issue with Easy-cla.

**Comment by windsonsea** [comment]
> @sftim could you please have a look at this PR? There seems to be an issue with Easy-cla.

All right. It's ok now.
/lgtm

**Pull request merged**